scripts/release/group_test.py

```diff
@@ -78,67 +92,68 @@ def run_pytest_jobs(pytest_jobs: dict, my_env: dict):
     test_coverage = {}
     covered_assets = []
 
-    if group_pre:
-        check_call(f"python {group_pre}", env=my_env, shell=True)
-
-    pytest_jobs = {}  # pytest job path -> assets coverage dict
-    with open(tests_dir / TEST_YML) as fp:
-        data = yaml.load(fp, Loader=yaml.FullLoader)
-        for job, job_data in data[test_group]['jobs'].items():
-            if 'pytest_job' in job_data:
-                pytest_jobs[tests_dir / job_data['pytest_job']] = job_data['assets']
+    with set_directory(tests_dir):
+        if group_pre:
+            check_call(f"python {group_pre}", env=my_env, shell=True)
```

Comment:
Should this be `python3` instead of `python`. The former is called on line 108 below.

Reply:
Thanks Louie. Updated